udata/core/jobs/forms.py

```diff
@@ -32,6 +32,12 @@ class PeriodicTaskForm(ModelForm):
     task = fields.StringField(_('Tasks'))
     enabled = fields.BooleanField(_('Enabled'))
 
+    def save(self, commit=True, **kwargs):
+        '''PeriodicTask is now dynamic and save behavior changed'''
```

Comment:
You're right, I'l link to the changeset in celerybeat-mongo et to the comming PR in mongoengine